app/build.gradle

```diff
@@ -20,10 +20,10 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:27.0.2'
-    testCompile 'junit:junit:4.12'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
```

Comment:
Let's use last stable instead

Reply:
Unfortunately there’s no 28th stable version of appcompat yet. And we can not (should not because of warning) use 27.* verstion of appcompat with 28 Android SDK